### ISSUE TITLE
We need to make the API easy to cite - here's how

### ISSUE THREAD

**Issue by statist7** [rcpch/digital-growth-charts-server#132]
[https://www.r-bloggers.com/2021/02/make-your-r-package-easier-to-cite/?utm_source=feedburner&utm_medium=email&utm_campaign=Feed%3A+RBloggers+%28R+bloggers%29](url)

The link is from R-bloggers but the principle is universal.

**Comment by pacharanero** [comment]
OK thanks, I will take a look at how we do it

**Comment by eatyourpeas** [comment]
Good idea @statist7 - This looks easy to do. Github needs to be linked to [Zenodo](https://about.zenodo.org/) see Github [docs](https://guides.github.com/activities/citable-code/)

**Comment by eatyourpeas** [comment]
Thinking maybe we should do this once we have finalised decision on decimal age correction.

**Comment by pacharanero** [comment]
Agreed - and I think the Project board team need to publish that decision in a Paeds journal so we can cite the change in our code really too

**Comment by eatyourpeas** [comment]
We now have a doi: 
[![DOI](https://zenodo.org/badge/261587883.svg)](https://zenodo.org/badge/latestdoi/261587883)
closing with cb08959.

**Comment by pacharanero** [comment]
I'm in the process of citing this software while finalising my editorial on the Incubator for ADC, but noticed that the Zenodo references a specific release of the Server, and the releases are behind the versions in [main.py](https://github.com/rcpch/digital-growth-charts-server/blob/v4.2.19/main.py)

So although this is not purely about Zenodo/DOI, it was noticed because of it.

* Releases may need to be updated to follow versions in [main.py](https://github.com/rcpch/digital-growth-charts-server/blob/v4.2.19/main.py)
* Zenodo presumably updates itself
* We need perhaps to have a SOP for how updates are done. Could be as simple as a Runbook in the forum, or we could go further and automate it further.

**Comment by pacharanero** [comment]
https://github.com/rcpch/rcpch-house-style/pull/1#pullrequestreview-5041893981

**Comment by pacharanero** [comment]
Done: CITATION.cff carries the Zenodo DOI (10.5281/zenodo.6303587), is version-synced by bump2version and pinned by test_api_version_is_consistent, and GitHub exposes the CFF metadata on the repository landing page. Implementers and researchers can cite the API directly.
